Remove hardcoded CFLAG -pipe
Remove hardcoded LDFLAG -g

Disable check for lrint to fix compile warning.

Patch by Kevin McCarthy <signals42@gmail.com>

--- configure.in
+++ configure.in
@@ -31,8 +31,8 @@
 AC_PROG_CC
 if test "$GCC" = yes
 then
-  CFLAGS="${CFLAGS} ${EXTRA_GCFLAGS}-pipe -Wall -Wno-unused -Wshadow -Wbad-function-cast -Wmissing-prototypes -Wstrict-prototypes -Wcast-align -Wcast-qual -Wpointer-arith -Wwrite-strings -Wmissing-declarations -Wnested-externs -Wundef -pedantic -fno-common"
-  LDFLAGS="${LDFLAGS} ${EXTRA_GLDFLAGS}-g"
+  CFLAGS="${CFLAGS} ${EXTRA_GCFLAGS} -Wall -Wno-unused -Wshadow -Wbad-function-cast -Wmissing-prototypes -Wstrict-prototypes -Wcast-align -Wcast-qual -Wpointer-arith -Wwrite-strings -Wmissing-declarations -Wnested-externs -Wundef -pedantic -fno-common"
+  LDFLAGS="${LDFLAGS} ${EXTRA_GLDFLAGS}"
 else
   CFLAGS="${CFLAGS} ${EXTRA_CFLAGS}"
   LDFLAGS="${LDFLAGS} ${EXTRA_LDFLAGS}"
@@ -65,7 +65,6 @@
         exit 1
 fi
 
-AC_CHECK_FUNCS(lrint)
 AC_CHECK_FUNCS(nanosleep)
 
 AC_CHECK_FUNCS(tgetent,have_tgetent=yes)
